secant/Utils/Logging/OSLogger.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  OSLogger.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 23.01.2023.
+//
+
+import Foundation
+import ZcashLightClientKit
+import os
+
+enum LoggerConstants {
+    static let sdkLogs = "sdkLogs"
+    static let tcaLogs = "tcaLogs"
+    static let walletLogs = "walletLogs"
+}
+
+var walletLogger: ZcashLightClientKit.Logger?
+
+enum WalletLoggerProxy {
```

Comment:
I would suggest to use shorter name here. It would be little bit easier to use. Also this is used only in wallet so that "Wallet" part of the name seems little bit redundant.

Reply:
updated